code/addons/highlight/package.json

```diff
@@ -64,7 +74,7 @@
   "bundler": {
     "entries": [
       "./src/index.ts",
-      "./src/highlight.ts"
```

Comment:
This change wasn't entirely necessary here, but seemed like a good idea to keep the addons consistent with each other.